components/portal/main/partials/header.html

```diff
@@ -18,10 +18,6 @@
     under the License.
 
 -->
-<!-- PRIORITY NOTIFICATIONS -->
-<notifications-bell slot="myuw-notifications" mode="priority" class="priority-gt-xs" ng-if="!guestMode" hide-xs>
```

Comment:
This removed the deprecated but still relied upon "priority notification" feature.